app/templates/events/view/tickets/index.hbs

```diff
@@ -95,11 +95,11 @@
       <tr>
         {{#link-to 'events.view.tickets.orders.index' tagName='td'
         class='ui green inverted segment link'}}
-          <span>{{t 'Completed'}}</span>
```

Comment:
@niranjan94 Opened a issue in the server https://github.com/fossasia/open-event-orga-server/issues/4410